pkg/api/types.go

```diff
@@ -283,6 +283,8 @@ type LivenessProbe struct {
 	Exec *ExecAction `json:"exec,omitempty"`
 	// Length of time before health checking is activated.  In seconds.
 	InitialDelaySeconds int64 `json:"initialDelaySeconds,omitempty"`
+	// Length of time before health checking times out.  In seconds.
+	TimeoutSecs int64 `json:"timeoutSecs,omitempty"`
```

Comment:
Thanks! Yeah, I did consider that. Sub-second is probably the better choice. It is just that http.client Timeouts are in seconds. Perhaps we should do our own timeout mechanism here.